ceph-dev-trigger/config/definitions/ceph-dev-trigger.yml

```diff
@@ -40,6 +40,32 @@
     builders:
       - conditional-step:
           condition-kind: regex-match
+          # only run on master or octopus, includes any wip branch
+          # that is using the release branch as part of its name
```

Comment:
how about emphasis the intention of this filter here? like

> build octopus and up on bionic and centos7, as we dropped the xenial support since octopus.